microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,100 @@
+To minimize the risks, users can disable the sharing of OS regional information by selecting **Never**. The **Never** option still allows websites to read regional information, but does block any specific OS regional customizations that the user has made from being exposed to website content. Instead, the browser's display language is used to derive a default region. 
+
+#### The Limited option (default)
+The default, recommended value is **Limited**. When using the **Limited** setting, the above risks are mitigated by ensuring that mismatched combinations of language and region aren't revealed to websites. The **Limited** option ensures that only valid combinations of the user's current browser display language preference and a given region value are used. **Limited** ensures that websites won't see a regional code anomaly (though it could still look unusual per geographic inference), and also ensures that for language processing, websites will at least provide a consistent language experience (not mixing languages in weird combinations). The **Limited** option compares the language part of the regional code that is derived from the browser’s display language, and the regional part coming from the OS setting, and if the language parts match and if the resulting regional code is a valid combination, will pass the result through to the website. If the language in the message doesn't match, the feature acts as if **Never** was selected (the regional information falls back to being derived from the browser’s display language). 
+
+Note: the **limited** value is similar to current Firefox behavior for exposing the regional code, as described in [Regional Preferences - Firefox Source Docs documentation (mozilla.org)](https://firefox-source-docs.mozilla.org/intl/locale.html#regional-preferences), but also offers the possibility to not share this information at all. 
+
+#### The Always option (not recommended)
+
+If **Always** is selected, the user is exposed to the full set of risks noted earlier: the potential to be easily signaled out for fingerprinting, and also to potentially see website language-processing problems and inconsistencies. However, the user's OS preferences will be made available to the website without any limitations or restrictions, to be sure their preferences are respected. 
+
+
+
+<!-- ====================================================================== -->
+## Policy Controls 
+A related new policy in Microsoft Edge, **ShareOSRegionThroughJavaScriptLocale**, provides a setting for corporate or enterprise-level default configurations.  This setting accepts an integer with the following potential values: 
+
+|Number| Description |
+|--------|-------------------------|
+| 1 | Limited | 
+| 2 | Always | 
+| 3 | Never | 
+
+In addition, an older policy, [ApplicationLocaleValue](/DeployEdge/microsoft-edge-policies#applicationlocalevalue), allows setting the Microsoft Edge locale and prevents users from changing it. 
```

Comment:
```suggestion
In addition, an older policy, [ApplicationLocaleValue](/DeployEdge/microsoft-edge-policies#applicationlocalevalue), sets the Microsoft Edge locale and locks it to prevent users from changing it. 
```